src/lib/components/pagination/Pagination.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Pagination extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+    this.pageNumbers = this.pageNumbers.bind(this);
+    this.renderPages = this.renderPages.bind(this);
+    this.getPage = this.getPage.bind(this);
+    this.nextPage = this.nextPage.bind(this);
+    this.prevPage = this.prevPage.bind(this);
+  }
+
+  getPage(pageNumber) {
+    this.props.onPageChange(pageNumber);
+  }
+
+  handleClick(event) {
+    event.preventDefault();
+    this.getPage(event.target.id);
+  }
+
+  render() {
+    return (
+      <ul className="pagination">
+        <li onClick={this.prevPage}>{'<'}</li>
+        {this.renderPages()}
+        <li onClick={this.nextPage}>{'>'}</li>
+      </ul>
+    );
+  }
+
+  renderPages() {
+    return this.pageNumbers().map(pageNumber => {
+      return (
+        <li id={pageNumber} onClick={this.handleClick} key={pageNumber}>
+          {pageNumber}
+        </li>
+      );
+    });
+  }
+
+  nextPage() {
+    if (this.props.currentPage === this.props.totalPages) {
```

Comment:
```
if (this.props.currentPage >= this.props.totalPages)
```
this might break if somebody send the wrong currentPage greater than totalPages in the first place